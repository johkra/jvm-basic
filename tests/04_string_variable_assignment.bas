string1$ = "Hello, world"
PRINT string1$
string2$ = string1$
PRINT string2$
string1$ = "Goodbye, world"
PRINT string1$
