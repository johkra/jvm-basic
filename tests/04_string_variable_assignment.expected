Hello, world
Hello, world
Goodbye, world
